semgrep-core/src/core/ast/AST_generic.ml

```diff
@@ -748,6 +748,7 @@ and special =
   | Op of operator
   (* less: should be lift up and transformed in Assign at stmt level *)
   | IncrDecr of (incr_decr * prefix_postfix)
+  | Require
```

Comment:
Can you add a comment explaining why we need this construct given we already have ImportFrom?
Maybe something like "This is similar to ImportFrom, but at the expression level; in languages such as Javascript or Ruby import/require can appear at any places, not just at the toplevel".